.eslintrc.js

```diff
@@ -4,8 +4,38 @@ module.exports = {
     node: true,
     jest: true,
   },
-  extends: ['plugin:vue/recommended', 'airbnb-base', 'prettier'],
-  plugins: ['prettier'],
+  extends: [
+    'airbnb-base',
+    /**
+     * Provides Airbnb's base JS .eslintrc as an extensible shared config.
+     * https://github.com/airbnb/javascript
+     */
```

Comment:
Kan vi flytte kommentarene til over det de kommenterer og gjøre dem til blokkommentarer (`/*`) i stedet for docstrings (`/**`)?